packet/client.go

```diff
@@ -14,17 +14,32 @@ import (
 	"github.com/pkg/errors"
 	"github.com/tinkerbell/boots/httplog"
 	tinkClient "github.com/tinkerbell/tink/client"
+	tw "github.com/tinkerbell/tink/protos/workflow"
 )
 
 type hardwareGetter interface {
 }
 
+type workflowGetter interface {
+	getWorkflowConnection() tw.WorkflowSvcClient
+}
+
+type workflow struct {
+	wsc tw.WorkflowSvcClient
+}
+
+func (w *workflow) getWorkflowConnection() tw.WorkflowSvcClient {
+	return w.wsc
+}
```

Comment:
Hey @Cbkhare I'm back to not wanting this interface here. When you and I talked offline you convinced me it would be useful for testing the `GetWorkflowsFromTink` and `DiscoverHardwareFrom*` methods. But it still was bugging me since the grpc client stuff is already an interface and it seemed like we could use a mock implementation of it to do the testing, but I was going to drop it.

But then I went and wanted to throw in some tests in #59 and went looking for mocking grpc clients in go and found that we can just mock the CacherClient (in my use case) for the tests. Lets go that route instead, PTAL at #59 for the nitty gritty.

Reply:
done, as discussed offline today, have removed the interface and added the call's as below 

in `packet/client.go`
```
import tw "github.com/tinkerbell/tink/protos/workflow"
# client struct {
..
workflowClient tw.WorkflowSvcClient //  instead of workflowGetter
}
// inside func NewClient
wg, err = tinkClient.TinkWorkflowClient()
if err != nil {
	return nil, errors.Wrap(err, "connect to tink")
}
```
in `packet/endpoints.go`
```
// c is the *client 
result, err = c.workflowClient.GetWorkflowContexts(context.Background(), &tw.WorkflowContextRequest{WorkerId: hwID}) 
```